genetic_algorithm/basic_string.py

```diff
@@ -0,0 +1,148 @@
+"""
+Simple multithreaded algorithm to show how the 4 phases of a genetic
+algorithm works (Evaluation, Selection, Crossover and Mutation)
+
+https://en.wikipedia.org/wiki/Genetic_algorithm
+
+
+Author: D4rkia
+"""
+
+from typing import List, Tuple
+import random
```

Comment:
```suggestion
import random
from typing import List, Tuple
```
We use `isort` to sort imports.